src/pages/markets/components/helper/_symbol.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { ReactElement } from 'react'
+import { SymbolContainer } from '../../static/style/_markets-style'
+import { Text } from 'components/elements'
+
+type SymbolProps = {
+    instruments_type: { src: string; text: ReactElement }[]
+}
+
+const Symbol = ({ instruments_type }: SymbolProps) => (
+    <React.Fragment>
+        {instruments_type.map((symbol, index) => (
+            <SymbolContainer key={index}>
```

Comment:
It's better not to use index as a key

Reply:
Unfortunately we have no alternative to use instead of index.